src/activities/morse_code/MorseCode.qml

```diff
@@ -370,12 +376,81 @@ ActivityBase {
 
         }
 
+        GCText {
```

Comment:
I would put an opacity of 1.0 instead of 0.9 to not be bothered by the elements behind (we can still see the question for example)